packages/offchain-resolver/src/persistance/migrations/20230825081456_init/migration.sql

```diff
@@ -0,0 +1,37 @@
+-- CreateTable
```

Comment:
If we're starting with a fresh db we can delete all previous migrations and just have a initial one